Extending/Property-Editors/Declaring-your-property-editor.md

```diff
@@ -0,0 +1,334 @@
+# Declaring your property editor
+
+Generally Umbraco supports two different ways to declare a property editor. Most commonly one would create a `package.manifest` file, and then use it for declaring one or more property editors. But as an alternative, property editors can also be declared using C#.
+
+A property editor consists of a number of mandatory properties, and some optional ones as well. As such, the outer JSON object for the property editor has the following properties:
+
+| Name | Type | Required | Description |
+| ---- | ---- | ---- | ---- |
+| `alias` | string | Yes | A unique alias that identifies the property editor. |
+| `name` | string | Yes | The friendly name of the property editor, shown in the Umbraco backoffice. |
+| `editor` | object | Yes |  |
```

Comment:
No description on the `editor` property? 😄

Reply:
Not sure why I missed that one. Added now 😉